docs/src/modules/java/pages/author-your-first-service.adoc

```diff
@@ -182,10 +186,119 @@ include::example$doc-snippets/src/main/java/com/example/api/ExampleEndpoint.java
 ----
 <1> Returning a record that gets serialized as JSON
 
+Restart the service and curl this command:
+
 [source, command line]
 ----
 curl localhost:9000/hello/hello/Bob/30
 ----
 
+== Explore the local console
+
+The Akka local console is a web-based tool that provides a convenient way to view and interact with your running service.
+
+=== Install the Akka CLI
+
+The local console requires that you install the Akka CLI first.
+
+Install the Akka CLI:
+
+[.tabset]
+Linux::
++
+--
+Download and install the latest version of `akka`:
+[source,bash]
+....
+curl -sL https://doc.akka.io/install-cli.sh | bash
+....
+
+If that fails due to permission issues, use:
+
+[source,bash]
+....
+curl -sL https://doc.akka.io/install-cli.sh | bash -s -- --prefix /tmp && \
+    sudo mv /tmp/akka /usr/local/bin/akka
+....
+
+You can pass options to the installer script with `-s --` e.g.:
+
+[source,bash]
+....
+curl -sL https://doc.akka.io/install-cli.sh | bash -s -- --prefix=$HOME --version=2.0.22 --verbose
+curl -sL https://doc.akka.io/install-cli.sh | bash -s -- -P $HOME -v 2.0.22 -V
```

Comment:
I don't like that we have the version number hardcoded here. The `cli/installation.adoc` is automatically updated/generated when we release a new cli (I hope/think), but there it is also 2.0.22. Latest is 3.0.6.

@ennru How does this work? Why do we need a version number at all, can't it install latest?

Reply:
I'm not sure if the version number can be completely removed from the command itself, but I replaced the hardcoded version number in the docs by using Asciidoc attribute substitution.